<launch>
    <arg name="folder" default="/home/nbore/Data/kitti/2011_09_26/2011_09_26_drive_0095_sync"/>

    <node pkg="velodyne_tracking" type="publish_velodyne_transforms" name="publish_velodyne_transforms" output="screen" respawn="false">
        <param name="cloud_stamps_path" value="$(arg folder)/velodyne_points/timestamps.txt"/>
        <param name="transforms_stamps_path" value="$(arg folder)/oxts/timestamps.txt"/>
        <param name="transforms_path" value="$(arg folder)/oxts/data/"/>
        <param name="cloud_path" value="$(arg folder)/velodyne_points/data/"/>
    </node>
</launch>
